### PR TITLE
List structure

### PR DESCRIPTION
**Pull request by riskmaris**
For this milestone:
1. I built a simple web page using Webpack and served by a Webpack dev server.
2. I updated my README file.
3. All my source files (index.html, index.js, and style.css) are located in the/src directory.
4. My distribution files are generated by Webpack and served by the Webpack dev server from the/dist folder.

### CODE REVIEW

File: src/styles.css [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+  justify-content: center;
+  flex-direction: column;
+  align-items: center;
+}
+
+.bi-journal-text {
+  font-size: 5rem;
+  color: rgb(29, 125, 185);
+  margin: 15px auto;
+}
```
Comment (binhussen) [inline src/styles.css]: - Kindly, stick to the given design to fit the [requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m1_list_structure.md) and have better UI 👍 
#### your Desgn 👇
Screenshot 2023-07-12 164751
#### Given Design 👇
Screenshot 2023-07-12 164816

File: webpack.config.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+//       },
+//     ],
+//   },
+//   plugins: [
+//     new HtmlWebpackPlugin({
+//       title: 'ToDo-List',
+//       filename: 'index.html',
+//     }),
+//   ],
+// };
```
Comment (binhussen) [inline webpack.config.js]: - Kindly, Do not commit old pieces of code as inline comments. They will make your project look messy. If you need to review a previous version of your code, you can always use git history. 👍

File: src/index.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
+// import './styles.css';
```
Comment (binhussen) [inline src/index.js:1]: - Great Job for styling your page with Webpack Style/CSS Loader, I think you commented on the style import here which leads to not applying the style, so uncomment it 👍

**Review by binhussen — CHANGES_REQUESTED** [review]
Hi @riskmaris , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

- Setuped Webpack ✔️
- Well Structured code ✔️
- Used GitHub flow ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: webpack.config.js (hunk 2 above)
Comment (rotimiazeez) [inline webpack.config.js]: As required by the previous reviewer, always ensure your codebase is clean. Kindly remove all commented codes. 😉

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi @riskmaris 😉, 

## Great work on making the changes 👏 
### You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


## Highlights 
* Good app UI ✔️

## Suggested changes
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me(@rotimiazeez) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by skyv26 — APPROVED** [review]
## Hi @riskmaris, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

## STATUS: APPROVED 🟢 



## Highlights
- Good **PR** title ✅ 
- Descriptive *PR* description ✅ 
- Implement the features ✅ 
- Nice *UI* design ✅ 
- Professional `README` template ✅ 
- Linters are green ✅ 
- Meaningful **Github** commits ✅ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## ⚠️  WARNING ⚠️ 
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

## Have any doubt ❓ 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @skyv26 in your question so I can receive the notification. You can also follow me on the below platforms

<table>
<tr>
<td>

<a href="https://github.com/skyv26">
   @skyv26
</a>

</td>
<td>

<a href="https://www.linkedin.com/in/skyv2022/">
   linkedin-skyv2022
</a>

</td>
<td>

<a href="https://twitter.com/vrma_aakash">
   @vrma_aakash
</a>

</td>
<td>

<a href="https://www.instagram.com/skybrel/">
   @skybrel
</a>

</td>
<td>

<a href="https://www.frontendmentor.io/profile/skyv26">
   @skybrel
</a>

</td>
</tr>
</table>

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**